test_app/router.py

```diff
@@ -12,3 +13,14 @@
 # this uses standard registration
 router.register(r'related_fields_test_models', views.RelatedFieldsTestModelViewSet)
 # intentionally not registering ResourceMigrationTestModel to test lack of URLs
+
+associative_router = AssociationResourceRouter()
+associative_router.register(
+    r'related_model',
+    views.RelatedFieldsTestModelViewSet,
+    related_views={
+        'teams': (views.TeamViewSet, 'more_teams'),
+        'user': (views.UserViewSet, 'users'),
+    },
+    basename='related_model',
+)
```

Comment:
I left the original router in place because the comments (on some of the items) made it seem like there are already some tests against the router that I didn't want to break. We can reevaluate it.

Reply:
sounds good, we can punt for later.